### PR TITLE
Quiz Pages

### PR DESCRIPTION
**Pull request by yogesh-**
Completed Quiz Pages with following :
- added responsiveness
- show score
- show rules of the quiz

Preview link - https://deploy-preview-3--code-quiz1.netlify.app/

image

image

### CODE REVIEW

**Review by chanchal-panpaliya — APPROVED** [review]
code looks clean with proper naming with good UI

File: src/pages/quiz/quiz.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  text-decoration: underline;
+}
+
+.question {
+  flex: 50%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border-right: 0.2rem solid #d0e90d;
```
Comment (Kothalkarpriya) [inline src/pages/quiz/quiz.css:29]: Colors can be defined in root and accessed as var

File: src/pages/quiz/quiz.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  display: flex;
+  flex: 50%;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 2rem;
+}
+
+.answer button:hover {
+  background: -webkit-linear-gradient(45deg, #d0e90d, #1536f1);
```
Comment (Kothalkarpriya) [inline src/pages/quiz/quiz.css:47]: here also

**Review by Kothalkarpriya — APPROVED** [review]
The code is clean and understandable. Added one suggestion

**Pull request merged**